text/0110-dynamic-components.md

```diff
@@ -6,266 +6,277 @@ updated_at: 2021-02-01
 
 ### General Invariants
 
-Note that the invariants defined below are not necessarily constraints on LWC in general but rather explicitly defined for the Lightning Platform use cases:
+The following invariants aren't necessarily constraints on LWC, but rather are explicitly defined for Lightning Platform use cases.
 
 - A dynamic import must always be statically verifiable.
-    - A dependency should never be discovered or learned of for the first time at runtime.
-    - Note that the ability to analyze is orthogonal to when a component is loaded.
+  - A dependency should never be discovered or learned of for the first time at run time.
+  - The ability to analyze is orthogonal to when a component is loaded.
 - It must not break the reactivity model or the general framework lifecycle.
-    - Once the component is loaded it should function within the lifecycle of the parent component.
-    - No special handling in user-land is needed.
+  - After the component is loaded, it should function within the lifecycle of the parent component.
+  - No special handling in user-land is needed.
 - It must work regardless of the algorithm, even if nothing is preloaded at all.
-    - The decision for when a dynamic dependency is loaded is orthogonal to its definition.
-    - We might have a preload mechanism based on MRU or on a specific context (ex.mobile) but the component should work regardless of the algorithm even if nothing is preloaded at all.
+  - The decision for when a dynamic dependency is loaded is orthogonal to its definition.
+  - We might have a preload mechanism based on MRU or on a specific context (for example, mobile) but the component should work regardless of the algorithm even if nothing is preloaded at all.
 
-You will notice that this follows the same pattern as what we have done for `@wire`: Bring the ability to understand and resolve dependencies at compile, build and design time.
+These invariants follow the same pattern that we use for `@wire`: the ability to understand and resolve dependencies at compile, build, and design time.
 
 ### API
 
-Below is the API and its ergonomics for dynamically loading a module. It has very similar mental model to the HTML `is` attribute proposal (which we can't use since Safari will not implement it in the foreseeable future): An arbitrary tag name on which you can assign any constructor.
+The API and ergonomics for dynamically loading a module has a similar mental model to the HTML `is` attribute proposal (which we can't use since Safari won't implement it in the foreseeable future): An arbitrary tag name on which you can assign any constructor.
 
-```json5
+```json
 {
-    "dynamicComponent": {
-        "loader": "@salesforce/loader",
-        "strict": true // only allow string identifiers
-    }
+  "dynamicComponent": {
+    "loader": "@salesforce/loader",
+    "strict": true // only allow string identifiers
```

Comment:
Nit: JSON doesn't allow for comments but JSON5 does

Reply:
Thank you, Eugene. I thought that was a typo -- bad assumption. I looked up the syntax highlighting for json5, and iit seems to be `javascript`: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L2551